ui/package.json

```diff
@@ -13,66 +13,66 @@
+    "@radix-ui/react-accordion": "^1.2.3",
     "@radix-ui/react-alert-dialog": "^1.1.6",
-    "@radix-ui/react-collapsible": "^1.1.2",
-    "@radix-ui/react-dialog": "^1.1.5",
+    "@radix-ui/react-collapsible": "^1.1.3",
+    "@radix-ui/react-dialog": "^1.1.6",
     "@radix-ui/react-dropdown-menu": "^2.1.6",
-    "@radix-ui/react-label": "^2.1.1",
-    "@radix-ui/react-popover": "^1.1.5",
-    "@radix-ui/react-progress": "^1.1.1",
+    "@radix-ui/react-label": "^2.1.2",
+    "@radix-ui/react-popover": "^1.1.6",
+    "@radix-ui/react-progress": "^1.1.2",
     "@radix-ui/react-radio-group": "^1.2.3",
     "@radix-ui/react-scroll-area": "^1.2.3",
-    "@radix-ui/react-select": "^2.1.5",
-    "@radix-ui/react-separator": "^1.1.1",
-    "@radix-ui/react-slot": "^1.1.1",
+    "@radix-ui/react-select": "^2.1.6",
+    "@radix-ui/react-separator": "^1.1.2",
+    "@radix-ui/react-slot": "^1.1.2",
     "@radix-ui/react-tabs": "^1.1.3",
-    "@radix-ui/react-tooltip": "^1.1.7",
-    "@react-router/node": "^7.1.4",
-    "@react-router/serve": "^7.1.4",
+    "@radix-ui/react-tooltip": "^1.1.8",
+    "@react-router/node": "^7.4.0",
+    "@react-router/serve": "^7.4.0",
     "ajv": "^8.17.1",
     "class-variance-authority": "^0.7.1",
     "clsx": "^2.1.1",
-    "cmdk": "^1.0.4",
+    "cmdk": "^1.1.1",
     "eslint-plugin-react": "^7.37.4",
-    "isbot": "^5.1.22",
-    "lucide-react": "^0.469.0",
-    "openai": "^4.81.0",
+    "isbot": "^5.1.25",
+    "lucide-react": "^0.483.0",
+    "openai": "^4.89.0",
     "react": "^19.0.0",
     "react-dom": "^19.0.0",
     "react-hook-form": "^7.54.2",
-    "react-router": "^7.1.4",
+    "react-router": "^7.4.0",
     "recharts": "^2.15.1",
     "smol-toml": "^1.3.1",
-    "tailwind-merge": "^2.6.0",
+    "tailwind-merge": "^3.0.2",
     "tailwindcss-animate": "^1.0.7",
     "tiktoken": "^1.0.20",
-    "typescript-eslint": "^8.22.0",
-    "uuid": "^11.0.5",
+    "typescript-eslint": "^8.28.0",
+    "uuid": "^11.1.0",
     "vite-plugin-wasm": "^3.4.1",
-    "zod": "^3.24.1"
+    "zod": "^3.24.2"
   },
   "devDependencies": {
-    "@react-router/dev": "^7.1.4",
+    "@react-router/dev": "^7.4.0",
     "@types/json-schema": "^7.0.15",
-    "@types/node": "^20.17.16",
-    "@types/react": "^19.0.8",
-    "@types/react-dom": "^19.0.3",
-    "@typescript-eslint/eslint-plugin": "^8.22.0",
-    "@typescript-eslint/parser": "^8.22.0",
-    "autoprefixer": "^10.4.20",
-    "eslint": "^9.21.0",
-    "postcss": "^8.5.1",
-    "prettier": "^3.4.2",
+    "@types/node": "^22.13.13",
+    "@types/react": "^19.0.12",
+    "@types/react-dom": "^19.0.4",
+    "@typescript-eslint/eslint-plugin": "^8.28.0",
+    "@typescript-eslint/parser": "^8.28.0",
+    "autoprefixer": "^10.4.21",
+    "eslint": "^9.23.0",
+    "postcss": "^8.5.3",
+    "prettier": "^3.5.3",
     "prettier-plugin-tailwindcss": "^0.6.11",
-    "tailwindcss": "^3.4.17",
-    "typescript": "^5.7.3",
-    "vite": "^5.4.14",
+    "tailwindcss": "^3.4.1",
```

Comment:
did you mean to downgrade TailWind (i.e. was `3.4.17` now `3.4.1`)?

Reply:
Good catch - not sure how this happened, maybe I accidentally deleted the character somehow or something went weird with the resolution? Anyways, fixed :)